### PR TITLE
S7 practice k

### PR DESCRIPTION
**Pull request by s7jackye**
Can you give a look?

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
+FROM httpd:2.4
+COPY ./public-html/ /usr/local/apache2/htdocs/
+WORKDIR /app
+
```
Comment (devopseasylearning) [inline Dockerfile:4]: ? the empty line

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+LABEL phone="800-989-5000" 
+LABEL address="14225 DevOps Success Rd, Houston, 
+LABEL email="info@devops.com"
+
+APP=DEVINE
+SECTION=V1 
+HOST=devopseasylearning.com
+PORT=5278 
+ZONE=PROD
+
```
Comment (devopseasylearning) [inline Dockerfile:15]: ? the empty line

File: Dockerfile (hunk 1 above)
Comment (olamyde) [reply Dockerfile:5]: Good to go

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,18 @@
+FROM httpd:2.4
+COPY ./public-html/ /usr/local/apache2/htdocs/
+
+WORKDIR /app
+
+# Set metadata for the container
+LABEL phone="800-989-5000" 
+LABEL address="14225 DevOps Success Rd, Houston, 
+LABEL email="info@devops.com"
+
```
Comment (s7rosine) [inline Dockerfile:10]: please remove line 10

**Review by s7rosine — COMMENTED** [review]
to make the Dockerfile looks professional

**Review by s7rosine — COMMENTED** [review]
but still have the comma at the end

**Review by devopseasylearning — APPROVED** [review]

**Review by sdjakou — APPROVED** [review]

**Review by NtBahati — APPROVED** [review]

**Review by EDEMOS80 — APPROVED** [review]

**Comment by devopseasylearning** [comment]
@s7jackye your merge is breaking the application please revert your change why did you raise the PR to production ?
I will be sending you and email with a zoom link for us to have a special 1:1 meeting with the Director of engineering and your manager

**Pull request merged**